backend/prisma/schema.prisma

```diff
@@ -8,11 +8,40 @@ datasource db {
 }
 
 model User {
```

Comment:
It also looks good for me 
Only one thing to mention. Next time you can use faker with factory pattern. It would satisfy us if we will need more then 1000 users for example 

Refs:
https://fakerjs.dev/
https://refactoring.guru/design-patterns/factory-method